### DISCUSSION TITLE
[Solved] Dark mode is not saved.

### DISCUSSION THREAD

**Discussion by Francy2007** [Difegue/Stylophone#58] [Q&A] [answered]
Good morning,
if possible I would like an answer from Difegue. I bought Stylophone on the Microsoft Store. When I set the Dark mode in Theme it is not saved after closing the app and therefore at the next start I have the System default mode. How can you save Dark mode?
Sincerely.

Vincenzo

**Comment by Difegue** [comment]
👋 Thanks for the report -- It appears theme switching broke in the 2.5.x UWP versions and doesn't save the change.  
I'll try to get an update with a fix shortly!

**Comment by Difegue** [comment, marked as answer]
V.2.5.5 is now out on the MS Store and should fix this alongside a few misc. crashes reported by AppCenter.  
Thanks again!

**Reply by Francy2007** [reply]
Everything's okay now. Thank you very much, very kind for having warned.
Sincerely.

Vincenzo
